politeiad/backendv2/tstorebe/plugins/comments/cmds.go

```diff
@@ -242,9 +242,15 @@ func (p *commentsPlugin) comments(token []byte, ridx recordIndex, commentIDs []u
 		c := convertCommentFromCommentAdd(v)
 		cidx, ok := ridx.Comments[c.CommentID]
 		if !ok {
-			return nil, fmt.Errorf("comment index not found %v", c.CommentID)
+			return nil, errors.Errorf("comment index not found %v", c.CommentID)
 		}
 		c.Downvotes, c.Upvotes = voteScore(cidx)
+		// Populate creation timestamp
+		c.CreatedAt, err = p.commentCreationTimestamp(c, cidx)
```

Comment:
Isn't this retrieving data that has already been fetched, i.e the `CommentAdd` records?  This call results in a tlog tree retrieval, so would be better from a performance standpoint if we don't need to make it.

Reply:
hrmmm we fetch the last version of each of the specified comments (see lines 201-217 above), to get the creation
timestamp of edited comments (version > 1) I needed to get the commentAdd of the first version of the comment, which **wasn't** fetched yet.

Reply:
Ok, makes sense.  I wanted to verify that this call is neccesary and it sounds like it is.